src/openrct2/Cheats.cpp

```diff
@@ -459,6 +459,8 @@ static void cheat_own_all_land()
         for (coords.x = min; coords.x <= max; coords.x += 32)
         {
             TileElement* surfaceElement = map_get_surface_element_at(coords);
+            if (!surfaceElement)
```

Comment:
Should have been `!= nullptr` as per Coding Style rules.

Reply:
Sorry, I work on multiple things so I can't quite remember every detail, I would have changed it of course. I wonder if clang-format can enforce this.

Reply:
Yeah, don't worry, two team members missed it as well.